odpf/optimus/core/v1beta1/job_spec.proto

```diff
@@ -102,14 +102,17 @@ message DeployJobSpecificationResponse {
 }
 
 message CreateJobSpecificationRequest {
+  reserved 3;
   string project_name = 1;
   string namespace_name = 2;
-  JobSpecification spec = 3;
```

Comment:
@sravankorumilli @Mryashbhardwaj should we keep `spec` with `deprecated ` flag or just use `reserved` sufficient?

Reply:
you are right, i too think marking it deprecated is a more logical approach.